prow/plugins/trigger/pull-request_test.go

```diff
@@ -424,6 +478,16 @@ func TestHandlePullRequest(t *testing.T) {
 			} else if !tc.ShouldComment && len(g.IssueCommentsAdded) > 0 {
 				t.Errorf("Expected no comments to github, but got %d", len(g.IssueCommentsAdded))
 			}
+			if tc.jobToAbort != nil {
+				pj, err := fakeProwJobClient.ProwV1().ProwJobs("namespace").Get(tc.jobToAbort.Name, metav1.GetOptions{})
+				if err != nil {
+					t.Fatalf("failed to get prowjob: %v", err)
+				}
+
+				if pj.Status.State != prowapi.AbortedState {
```

Comment:
Would you mind to also verify here that the job is not set to `Completed`? It seems I didn't think of adding a test for that when I built that, even though its important behavior

Reply:
Thank you for the review. Done.